source/slang/check.cpp

```diff
@@ -1894,6 +1894,15 @@ namespace Slang
                     numThreadsAttr->y          = (int32_t) yVal->value;
                     numThreadsAttr->z          = (int32_t) zVal->value;
                 }
+                else if (auto bindingAttr = attr.As<GLSLBindingAttribute>())
+                {
+                    SLANG_ASSERT(attr->args.Count() == 2);
+                    auto binding = checkConstantIntVal(attr->args[0]);
+                    auto set = checkConstantIntVal(attr->args[1]);
```

Comment:
I assume that the case where the second argument is left off is auto-magically handled by the default parameter on the declaration? We probably need a test case to cover that since I wouldn't trust myself to have implemented default parameters on attributes correctly.

Reply:
In the definition in the core.meta.slang - you can define a default value which I do as zero, and I believe this works, although having a test makes sense.